test/s3_utils.py

```diff
@@ -0,0 +1,64 @@
+from botocore.exceptions import ClientError
+from cloudpathlib import S3Path
+import json
+
+
+def remove_bucket(s3, bucket_name: str) -> None:
+    """
+    Remove an S3 bucket.
+    """
+    try:
+        s3.delete_bucket(Bucket=bucket_name)
+    except ClientError as e:
+        print("The bucket does not exist: {}".format(e))
+
+
+def remove_objects(s3, bucket_name: str) -> None:
+    """
+    Remove all the contents of an s3 bucket.
+    """
+    try:
+        bucket = s3.Bucket(bucket_name)
+        bucket.objects.all().delete()
+    except ClientError as e:
+        print("The bucket does not exist: {}".format(e))
```

Comment:
Just a question - do we want the behaviour of these "remove" functions to swallow the error - it might be good if the process fails so that we know there are orphaned resources in AWS?